extensions/delivery-tracker/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Delivery Tracker Changelog
+
+## [Initial Release] - {PR_MERGE_DATE}
+
+The initial release.  Has support for...
+- UPS.
+- FedEx.
+- USPS.
```

Comment:
Can we make this look better 🙂

Reply:
Happy to!

Reply:
I improved it slightly.  Let me know if you're looking for more or not.